index.js

```diff
@@ -144,7 +223,25 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
       { "James and the Giant Peach": "91%" },
     ];
  */
-function getRottenTomatoesScoreByMovie() {}
+function getRottenTomatoesScoreByMovie(movies) {
+  if (!movies.length){
+    throw "error - there are no movies."
+  }
+
+  const rottenTomatoesScoreByMovie = movies.map((movie) => {
+    // console.log(movie.ratings[1].value)
+    // console.log(movie.ratings[1].value) // Not correct values of "Rotten Tomatoes" - this is the first index (which changes)
+  })
+  let movieRating = movies.find((rating) => {
+    if (rating.source === "RottenTomatoes"){
+      return rating.value;
```

Comment:
instead of just returning the Rotten Tomatoes score (which is what movieRating.value should point to) the problem is asking us to return an **object** with the movie.title as the key, and the Rotten Tomatoes score as the value.

```suggestion
      return {[movie.title] : movieRating.value}
```